Applications/spire/source/charting/chart_view.cpp

```diff
@@ -115,7 +116,7 @@ void ChartView::paintEvent(QPaintEvent* event) {
   painter.setPen(Qt::white);
   painter.drawLine(m_x_origin, 0, m_x_origin, m_y_origin);
   painter.drawLine(0, m_y_origin, m_x_origin, m_y_origin);
-  if(m_x_range <= m_x_axis_step || m_y_range <= m_y_axis_step) {
+  if(m_x_range < ChartValue(0) || m_y_range < ChartValue(0)) {
```

Comment:
What if it is equal to 0?

Reply:
Then it would also be invalid. If the region's top and bottom were equal then the chart wouldn't be able to zoom out.

Reply:
Okay then you need to prevent that scenario by ensuring the zoom never gets to that point.

Reply:
The debugger is showing me after scrolling close enough in, the width_change and height_change values don't change anymore between scrolls, they just stay the same. My suspicion is that adding 8.4^-16 to the height, for instance, isn't changing anything.

So I guess there is a technical limit to how much the region can be zoomed in, but is this good enough to rely on?